File: Logical/2.rb
puts("아이디를 입력해주세요")
input_id = gets.chomp()
puts("비밀번호를 입력해주세요")
input_pw = gets.chomp()
real_id = "egoin"
real_pw = "11"
if real_id == input_id
  if real_pw == input_pw
    puts("Hello!")
  else
    puts("잘못된 비밀번호입니다")
  end
else
  puts("잘못된 아이디입니다")
end

File: Logical/2.py
input_id = input("아이디를 입력해주세요\n")
input_pw = input("비밀번호를 입력해주세요\n")
real_id = "egoing"
real_pw = "11"
if real_id == input_id:
    if real_pw == input_pw:
        print("Hello!")
    else:
        print("잘못된 비번입니다")
else:
    print("잘못된 아이디입니다")
